src/datanode/src/instance/grpc.rs

```diff
@@ -150,10 +150,17 @@ impl Instance {
         }
     }
 
-    async fn execute_logical(&self, plan_bytes: Vec<u8>) -> Result<Output> {
-        let logical_plan_converter = DFLogicalSubstraitConvertor::new(self.catalog_manager.clone());
-        let logical_plan = logical_plan_converter
-            .decode(plan_bytes.as_slice())
+    async fn execute_logical(&self, plan: LogicalPlanProto) -> Result<Output> {
+        let table = self
+            .catalog_manager
+            .table(&plan.catalog, &plan.schema, &plan.table)
+            .context(CatalogSnafu)?
+            .context(TableNotFoundSnafu {
+                table_name: format!("{}.{}.{}", &plan.catalog, &plan.schema, &plan.table),
+            })?;
```

Comment:
`DFLogicalSubstraitConvertor` keeps a `CatalogManager` and will do this when translating `TableScan` plan. So I think these lines are redundant.